src/executors/default.yml

```diff
@@ -1,8 +1,8 @@
-description: "The official CircleCI PHP Docker image."
+description: "The official CircleCI CIMG PHP Docker image."
 parameters:
   tag:
-    description: "The `circleci/php` Docker image version tag."
+    description: "The `cimg/php` Docker image version tag."
     type: string
     default: "latest"
```

Comment:
There is no "latest" tag of the cimgs to my knowledge. @felicianotech would "stable" work here?

Reply:
I don't believe so. Because of the nature of PHP development (everyone is using one version or another), I think it might be safe to make this a required parameter, rather than making the decision for them.